### DISCUSSION TITLE
4.4.173 Downloaded direct by ci.appveyor.com finally **(Inoffizial Release/nightly )**

### DISCUSSION THREAD

**Discussion by blackcrack** [Open-Shell/Open-Shell-Menu#1109] [General]
Download today befor 2-4 Minutes, add a info.txt , compressed with Bandizip 6.26 with Windows 7,
rename the Zip correctly, Install it and therewith test the packet and upload it here as Attachment.

4.4.173
Various single-click option fixes
Fixed single-click option ignoring split arrows
Fixed Apps folder being clickable
6 days ago by thisismy-github (committed by Ibuprophen)
master 64259f73
5 days ago in 6 min 38 sec

OpenShellSetup_4_4_173.exe MD5 : FF971A96BB8D3704F97EDEDEE62EB8D3
OpenShellPDB_4_4_173.7z    MD5 : 064BF8FCA2686EDAE43F8C10CB80AB3E

[OpenShellSetup_4_4_173_2022-08-12.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/9386401/OpenShellSetup_4_4_173_2022-08-12.zip) 25.179 KB on locally NTFS  

4.4.172
Various single-click option fixes
Fixed single-click option ignoring split arrows
Fixed Apps folder being clickable
6 days ago by thisismy-github (committed by Ibuprophen)
master 64259f73

4.4.171
Fix corrupted icons in second column (#1088)
When using skin with a different second column icon size, all of the
icons in it would become blank.
Because m_bTwoColumns is set after AddStandardItems, it is still false
in the column break check. As a workaround, use the value it's assigned
later in InitWindowInternal instead.
Fix Open-Shell/Open-Shell-Menu#980
8 days ago by among-us-official (committed by GitHub)
master b89aaed7

best your'
Blacky

**Comment by ge0rdi** [comment]
Nightly builds are now on Github releases page:
https://github.com/Open-Shell/Open-Shell-Menu/releases

**Reply by blackcrack** [reply]
![grafik](https://user-images.githubusercontent.com/7925717/185798225-756f3d6f-b338-468e-ab31-4590b0857379.png) ![grafik](https://user-images.githubusercontent.com/7925717/185798226-97a59770-eff6-499f-be9e-03ade3a38c4d.png) ![grafik](https://user-images.githubusercontent.com/7925717/185798228-ab8fd6c3-15dc-4cbc-9058-8f0acb176088.png)
